ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/EquipmentUpgradeFilterViewModel.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using CommunityToolkit.Mvvm.ComponentModel;
+using ElectronicObserver.Common;
+using ElectronicObserver.Window.Tools.EquipmentUpgradePlanner.Helpers;
+
+namespace ElectronicObserver.Window.Tools.EquipmentUpgradePlanner;
+
+public class EquipmentUpgradeFilterViewModel : ObservableObject
+{
+
+	public bool DisplayFinished { get; set; } = true;
+
+	public List<EquipmentUpgradeFilterDayViewModel> Days { get; set; } = new();
+
+	public DayOfWeek? SelectedDay { get; set; } = null;
```

Comment:
Think it might be best to just initialize this to today.

Reply:
I'll make it today but JST